pkg/bitcoin/address_test.go

```diff
@@ -192,328 +80,250 @@ func TestEncodeAddress(t *testing.T) {
 	}
 
 	tests := []struct {
-		name    string
-		request *pb.EncodeAddressRequest
-		want    *pb.EncodeAddressResponse
-		wantErr *status.Status
+		name        string
+		publicKey   []byte
+		encoding    AddressEncoding
+		chainParams ChainParams
+		want        string
+		wantErr     error
```

Comment:
If you're just gonna check the `error.Error()` message, just use a string (will be `""` instead of `nil` if not specified), it will avoid instanciating raw errors in test declaration and then calling `.Error()` to check the result

Reply:
I generally agree, but in this case, I prefer using the `error` type since I intend to add proper publicly-exported error types upstream, which is currently using `errors.New(...)`.